### ISSUE TITLE
Raw message: could not convert input to UTF-8

### ISSUE THREAD

**Issue by mxmehl** [astroidmail/astroid#370]
Quite often, when I want to see a mail's raw message, I can just see for example:

```
Filename: /home/max/Mails/FSFE/INBOX/cur/1500909747_0.32272.bistromath,U=4628,FMD5=7e33429f656f1e6e9d79b29c3f82c57e:2,S

Error: Could not convert input to UTF-8.
```

But the file looks perfectly fine. `file` also labels the file as `UTF-8 Unicode text`.

If necessary, I can send the file privately.

**Comment by gauteh** [comment]
Max writes on juli 24, 2017 17:38:
> If necessary, I can send the file privately.

That'd be good :)

**Comment by gauteh** [comment]
Should be fixed on master, note that #368 also is merged.
